perf/CMakeLists.txt

```diff
@@ -5,6 +5,11 @@ add_subdirectory(lua)
 find_package(benchmark QUIET)
 if (NOT ${benchmark_FOUND})
     message(AUTHOR_WARNING "Google Benchmark library was not found")
+    set(MSG "Target test-c-perf is dummy, Google Benchmark library was not found")
```

Comment:
Minor: I don't think this variable is required here. IMHO it only makes the whole part less readable. Feel free to ignore.

Reply:
String with message is used below two times, so variable is requied.